src/App.js

```diff
@@ -38,35 +38,49 @@ const UserIsAuthenticated = UserAuthWrapper({
  * Expose App base component. Handle all routes
  */
 
+const buildRoutes = (store, onLogPageView, defaultRoute, features, userManager) => {
+
+  const getComponent = component => features.authEnabled ? UserIsAuthenticated(component) : component;
+
+  const router = (
+    <Router history={browserHistory} onUpdate={onLogPageView}>
+      <Redirect from="/" to={defaultRoute} />
+      <Route path="login" component={Login} />
+      <Route path="about" component={About} />
+      <Route path="/callback" component={CallbackPage} />
+      {features.trust !== false ?
+        <div>
+          <Route path="search-creator" component={SearchCreator} />
+          <Route path="saved-searches" component={SeeAllSearches}/>
+          <Route path="saved-search/:id" component={SearchDetail} />
+          <Route path="edit-search/:id" component={SearchEditor} />
+        </div>
+      : null}
+      {features.ask ?
+        <div>
+          <Route path="forms" component={getComponent(FormList)} />
```

Comment:
This component you're describing (if I'm reading this correctly) is essentially what `UserAuthWrapper` from [redux-auth-wrapper](https://github.com/mjrussell/redux-auth-wrapper) does. My fix is only a couple of lines and doesn't require a different component. I tried just changing the `<div>` that wraps the Ask routes to a `<Route>` and putting the component there, but I didn't know what component to pass into `UserIsAuthenticated()`